docs/references/chrome-extension/create-clerk-client.mdx

```diff
@@ -0,0 +1,163 @@
+---
+title: '`createClerkClient()`'
+description: "`createClerkClient()` allows Chrome Extension developers to access a user's session information or get a token in a service worker."
+---
+
+The `createClerkClient()` helper initializes a new Clerk instance on demand and refreshes the session token if there is a valid, signed-in user. It can be used in a [content script](https://developer.chrome.com/docs/extensions/develop/concepts/content-scripts) or a [background service worker](https://developer.chrome.com/docs/extensions/develop/concepts/service-workers/basics) to access a user's information or session token.
```

Comment:
```suggestion
The `createClerkClient()` helper initializes a new Clerk instance on demand and refreshes the session token if there is a valid, signed-in user. It can be used in a [background service worker](https://developer.chrome.com/docs/extensions/develop/concepts/service-workers/basics) to access a user's information or session token.
```